pkg/remote/terraform/provider.go

```diff
@@ -217,3 +221,7 @@ func (p *TerraformProvider) Cleanup() {
 		client.Close()
 	}
 }
+
+func (p *TerraformProvider) GetConfig() TerraformProviderConfig {
+	return p.Config
```

Comment:
Why don't you use this getter but created it ?

Reply:
I created it to implement it in the ResourceReader interface but it's no longer needed